config/keys.go

```diff
@@ -209,4 +209,5 @@ const (
 	TracingSampleRateKey                               = "tracing-sample-rate"
 	TracingExporterTypeKey                             = "tracing-exporter-type"
 	TracingHeadersKey                                  = "tracing-headers"
+	RuntimeStatePathKey                                = "runtime-state-path"
```

Comment:
I think we typically differentiate between `file` and `dir` explicitly in the name... So I think this should probably be`runtime-state-file` rather than `runtime-state-path`... I guess with the exception of `ipcs-path`...